Week1/homework/style.css

```diff
@@ -1 +1,55 @@
-/* add your styling here */
\ No newline at end of file
+/* add your styling here */

```

Comment:
Your styling works, but it could be improved. Don't forget to test your layouts on different screen sizes - mobile users will visit your websites too! Nice work on using `flex` - try challenge yourself further and try out  `grid` - which is perfect for laying out your grid of books.